hotsos/core/config.py

```diff
@@ -112,6 +112,25 @@ def __init__(self):
                                         'log files, this is used to limit the '
                                         'logrotate history in days.'),
                            default_value=7))
+        self.add(ConfigOpt(name='allow_constraints_for_unverifiable_logs',
+                           description=('Search constraints use a binary '
+                                        'search that sometimes needs to '
+                                        'search backwards to find a last '
+                                        'known good line i.e. in log files '
+                                        "that contain lines that don't start "
+                                        'with a timestamp we treat those as '
+                                        'unverifiable and so have to find the '
+                                        'most recent verifiable line which '
+                                        'can be ahead or behind the current '
+                                        'position. Seeking backwards seems to '
+                                        'force a SEEK_SET in the kernel which '
```

Comment:
I understand the issue - but I doubt it's the *kernel* setting SEEK_SET (I suspect it's somewhere in the Python lib).

Reply:
yeah I can reword this as its not entirely correct, basically what I see is that if i am at pos 100002 and want to seek(SEEK_SET, 100001) it will seek from the start of the file which may be intentional but if i do seek(SEEK_CUR, -1) it also forces to SEEK_SET and starts from 0 (which i can see in strace).

Reply:
ive updated the description fwiw